src/adapter/src/session.rs

```diff
@@ -665,11 +661,6 @@ impl<T: TimestampManipulation> Session<T> {
         }
     }
 
-    /// Returns whether the current session is a superuser.
-    pub fn is_superuser(&self) -> bool {
-        self.user.is_superuser()
-    }
-
```

Comment:
This function will start being useful for when the RBAC checks are implemented. It can be re-written as `self.user().is_superuser()`. I could also just re-implement it when the time comes.